main.go

```diff
@@ -62,10 +62,19 @@ import (
 	"github.com/Icinga/icingadb/prometheus"
 	"github.com/Icinga/icingadb/supervisor"
 	log "github.com/sirupsen/logrus"
+	"os"
+	"os/signal"
 	"sync"
+	"syscall"
 )
 
 func main() {
+	{
+		ch := make(chan os.Signal, 1)
+		signal.Notify(ch, syscall.SIGTERM, syscall.SIGINT)
+		go handleSignal(ch)
+	}
```

Comment:
Does this really need to be its own scope? Couldn't you just move all of this into the handleSignal function?

Reply:
1. We don't need `ch` on the stack.
2. I prefer to capture as less as possible in functions/closures, on principle.